forge-gui/res/cardsfolder/upcoming/the_day_of_the_doctor.txt

```diff
@@ -0,0 +1,17 @@
+Name:The Day of the Doctor
+ManaCost:3 R W
+Types:Enchantment Saga
+K:Saga:4:DBDig,DBDig,DBDig,DBChoose
+SVar:DBDig:DB$ DigUntil | Defined$ You | Valid$ Card.Legendary | FoundDestination$ Exile | RevealedDestination$ Exile | RememberRevealed$ True | ImprintFound$ True | SubAbility$ DBEffect | SpellDescription$ Exile cards from the top of your library until you exile a legendary card. You may play that card for as long as CARDNAME remains on the battlefield. Put the rest of those exiled cards on the bottom of your library in a random order.
+SVar:DBEffect:DB$ Effect | StaticAbilities$ MayPlay | RememberObjects$ Imprinted | Duration$ UntilHostLeavesPlay | SubAbility$ DBRestRandomOrder
+SVar:MayPlay:Mode$ Continuous | Affected$ Card.IsRemembered+nonLand | MayPlay$ True | EffectZone$ Command | AffectedZone$ Exile | Description$  You may play that card for as long a CARDNAME remains on the battlefield.
+SVar:DBRestRandomOrder:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered+IsNotImprinted | Origin$ Exile | Destination$ Library | LibraryPosition$ -1 | RandomOrder$ True | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | ClearImprinted$ True
+SVar:DBChoose:DB$ ChooseCard | Defined$ You | MinAmount$ 0 | Amount$ 3 | Choices$ Doctor | ChoiceTitle$ Choose up to three doctors | SubAbility$ DBGenericChoice
```

Comment:
I think you need some (all?) SpellDescription here for the Saga Card Detail

Reply:
Yeah spelldescription is all messed up here. But I don't know how to fix this without the generic choice 🤔

Reply:
Oh, is the Saga also grabbing the descriptions for the choices when it builds the detail?

Reply:
Yup, and considering the description of line 14 isn't present in the oracle (but necessary for the player when making the choice) It's a mess

Reply:
I have no idea what you guys are talking about:
![grafik](https://github.com/Card-Forge/forge/assets/8506892/35c20200-db1c-4fc2-80f1-fe83199720ef)

Reply:
Yeah, that's the output I was expecting @tool4ever – so the whole description needs to be in line 10

Reply:
"doctors" should be in uppercase in the prompt.